### ISSUE TITLE
[MocaTree]  Improvements for working with MocaTrees

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eMoflon/emoflon#208]
```
1.  Next edges are created by XML and ANTLR parsers per default between node 
children but not between attribute children.  It should be possible to disable 
this with an optional flag for backward compatibility.

2.  We should step up to ANTLR and ST 4.  This means that we have to switch 
from tree grammars to "model adaptors", and build a generic model adaptor for 
MocaTrees.
```

Original issue reported on code.google.com by `tonyAnjo...@googlemail.com` on 8 Oct 2014 at 9:40

**Comment by anthonyanjorin** [comment]
We could also just switch to Xtend instead of (2).

**Comment by anthonyanjorin** [comment]
This has been merged into #567
